documentstore_migracao/utils/convert_html_body.py

```diff
@@ -321,6 +322,20 @@ def transform(self, data):
             logger.debug("FIM: %s" % type(self).__name__)
             return data, xml
 
+    class FixATagPipe(plumber.Pipe):
+        def _change_src_to_href(self, node):
+            href = node.attrib.get("href")
+            src = node.attrib.get("src")
+            if (href is None or len(href) == 0) and src is not None and len(src) > 0:
```

Comment:
por que não:
```
python
if not href and src:
```

Reply:
@robertatakenaka Eu estava recebendo alguns Warnings na versão 3.7 para evitar de usar assim e passei a não usar. Não consigo encontrar a referência agora mas posso procurar.

Reply:
Consultei a PEP8 e não há especificamente alguma recomendação sobre isso, somente um alerta de garantir que `if obj` é o mesmo que `if obj is not None`.

https://www.python.org/dev/peps/pep-0008/#programming-recommendations